File: Nível 2/1159 - Soma de Pares Consecutivos/main.lua
while true do
    valor = tonumber(io.read())

    if valor == 0 then
        break
    else
        calculo = valor % 2

        if calculo ~= 0 then
            valor = valor + 1
        end

        formula = (valor) + (valor + 2) + (valor + 4) + (valor + 6) + (valor + 8)
        print(formula)
    end
end

File: Nível 2/1159 - Soma de Pares Consecutivos/main.py
while True:
    valor = int(input("A: "))

    if valor == 0:
        break
    else:
        calculo = valor % 2

        if calculo != 0:
            valor += 1
        
        formula = (valor) + (valor + 2) + (valor + 4) + (valor + 6) + (valor + 8)

        print(formula)